edit_verify.php

```diff
@@ -0,0 +1,19 @@
+<?php
+require_once 'functions.php';
+$validateFields = validateFields($_POST);
+$validateStats = validateStats($_POST);
+if($validateFields === false){
+    header('location: char_edit.php?error=All fields required');
+}
+if($validateStats === false){
+    header('location: char_edit.php?error=Stats must be below 20');
+}
+$charname = filter_var($_POST['charname'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
+$class = filter_var($_POST['class'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
```

Comment:
You're also repeating these two lines now, might be worth creating a function that does this for you